controller/signUpController.js

```diff
@@ -0,0 +1,81 @@
+const User = require('../models/user')
+const jwt = require('jsonwebtoken')
+const bcrypt = require('bcrypt')
+const secretKey = require('../config/secretKey')
+const dotenv = require('dotenv')
+const http = require('http-status-codes')
+const email_verify = require('./emailVerify')
+const statusMsg = require('../config/statusMsg')
+const SALTING = 10
+const { validationResult } = require('express-validator/check')
+
+dotenv.config({
+    path: './config/.env'
+})
+
+exports.create = async (req, res, next) => {
+    const errors = validationResult(req);
+    if (!errors.isEmpty()) {
+        return res.status(http.UNPROCESSABLE_ENTITY).json({ errors: errors.array() });
+    }
+    let hash = await bcrypt.hash(req.body.password, SALTING)
+    const access_token = await jwt.sign({ email: req.body.email }, secretKey.token.key, { expiresIn: process.env.access_token_exp })
```

Comment:
@AwaleRohin what if exception raises in jwt.sign.  how will you handle? never handle the generic errors.